etc/rsync/rsyncd.conf.j2

```diff
@@ -0,0 +1,61 @@
+log file = /var/log/rsyncd.log
+pid file = /var/run/rsyncd.pid
+lock file = /var/lock/rsyncd
+uid=root
+gid=root
+{% if 'logserver-slave' in group_names %}
+
+[openlab_logserver]
+comment = sync zuul logs
+path = /srv/static/logs/
+use chroot=yes
+read only = no
+max connections=10
+auth users = root
+secrets file = /etc/rsyncd/rsyncd.pass
```

Comment:
These lines are same of all the sections, is it able to common config section and works for all items?

Reply:
yes, it is not need to distinguish, I have tested